res/controllers/Pioneer-DDJ-SB.midi.xml

```diff
@@ -236,15 +236,6 @@
                     <normal/>
                 </options>
             </control>
-            <control>
-                <group>[PreviewDeck1]</group>
-                <key>play</key>
-                <status>0x96</status>
-                <midino>0x66</midino>
-                <options>
-                    <normal/>
-                </options>
```

Comment:
what did this button previously do?

Reply:
This button used to play the selected track in the preview deck, but because there was no custom behaviour to pause the preview when clicking the button a second time the only way to stop the preview was to stop it manually by clicking the pause button of the preview deck.

Reply:
Right, sorry I phrased my question wrong, what does the button do now? It seems like it doesn't do anything?

Reply:
Ah my bad, I didn't understand it that way. Now this behaviour is handled by the `rotarySelectorClick` midi mapping like so:

```xml
            <control>
                <group>[PreviewDeck1]</group>
                <key>LoadSelectedTrackAndPlay</key>
                <key>PioneerDDJSB.rotarySelectorClick</key>
                <status>0x96</status>
                <midino>0x41</midino>
                <options>
                    <normal/>
                    <script-binding />
                </options>
            </control>
```

(lines 745-753 of [Pioneer-DDJ-SB.midi.xml](https://github.com/mixxxdj/mixxx/pull/12280/files#diff-a94fec3379678bb49e2e22e882e73cba73a94b6b548a05f6c56969604806808f))

Reply:
Right, its good to know that no functionality was lost thanks to it being implemented in JS, but what I'm really interested in is whether the button is now unused. I can't spot it being handled anywhere so it just dead? What button on the controller is this exactly? I can't really guess just by looking at pictures.

If the button is completely unused now, that seems like a waste of valuable control-surface. Does that make sense?

Reply:
You're right I was missing something. According the [MIDI mappings sheet](https://www.pioneerdj.com/-/media/pioneerdj/software-info/controller/ddj-sb/ddj-sb_list_of_midi_messages_e.pdf) this button is actually the "BACK" button below the browser rotary selector. Thing is according to the manual its usage is to go back from browsing tracks into a crate/playlist to browsing the side panel. But in Mixxx browsing this panel is done using shift+rotary selector so this feature doesn't really have a proper usage. Do you have an idea of what relevant action could be done with this button instead?

Looking at the manual the shift+click action for this button is switch between panels in Serato DJ, so I'm thinking it could be used to toggle between showing/hiding the decks in Mixxx (what happens when you press the spacebar on your keyboard) but I'm not sure how to implement this, could you please guide me a bit?

<img width="442" alt="image" src="https://github.com/mixxxdj/mixxx/assets/19548672/d62b7e92-d0c0-4cbb-a9d8-bb2d6cb93d6f">

Reply:
In general it's recommended to make the buttons works as labeled. No need to reproduce special / unintuitive behaviour of other apps.
* "Back" would be [[Library],MoveFocusForward](https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[Library]-MoveFocusForward), equivalent to pressing Tab on the keyboard. (Shift +press would be ..Backward (Shift + Tab)).
This allows to move focus between sidabar, tracks table and searchbar (and its Clear button)[1]. Focussing the search bar becomes relevant in Mixxx 2.4 where the searchbar is turned into a combobox that holds previous search queries.
* Rotating the encoder ([[Library],MoveVertical](https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[Library]-MoveVertical)) would then allow selecting a query, top-most item is 'clear'.
* Pressing the rotary should be [`[Library],GoToItem`](https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[Library]-GoToItem)
* Shift + press could be whatever you like (preview selected track). You may also implement Shift + longpress to have one more action (maximize library)

[1] we should discuss if we skip the Clear button in case a MoveFocus controls is used, unrelated to this PR

Reply:
Thanks for the feedback, I'll work on this. Although the alternative action for the BACK button is labelled as PANEL (see picture) so I think it would make sense for it to control toggling UI components when shift clicking, what do you think?

Reply:
maybe shift+back could be [`[Skin],show_maximized_library`](https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls#control-[Master]-maximize_library) (we renamed it, the manual is outdated in that in that regard).

Reply:
Sure, no objections.
Use `[Skin],show_maximized_library` if that's what you need, and maybe mention in the manual that this control can be customized?